docs/en/userguide/01_User_manual/06_Remember_trusted_devices.md

```diff
@@ -0,0 +1,30 @@
+title: Remember trusted devices
+summary: Steps to stay signed in on a trusted device
+
+# Remember trusted devices
+
+In order to avoid repeat requests to log in with multi-factor authentication through a device you know to be secure, it is possible to assign your device as a 'trusted device' at the log-in screen.
```

Comment:
```suggestion
In order to avoid repeat requests to log in with Multi-factor authentication (MFA) through a device you know to be secure, it is possible to assign your device as a 'trusted device' at the log-in screen.
```